pytket/extensions/qiskit/backends/aer.py

```diff
@@ -127,7 +127,7 @@ def rebase_pass(self) -> BasePass:
         )
 
     def _arch_dependent_default_compilation_pass(
-        self, arch: Architecture, optimisation_level: int = 2
+        self, arch: Architecture, optimisation_level: int = 2, timeout: int = 1000
```

Comment:
This looks to me like something that is overwriting the default value in tket. Maybe we can set the default value to none and not append the value if it is not set? (same below) ?

Reply:
Ah yeah I think you're right. I will change that